tests/unit_test/job_config/fed_job_test.py

```diff
@@ -14,10 +14,10 @@
 
 import pytest
 
+from nvflare import FedJob
```

Comment:
either way works, but for this PR, maybe don't change this file, we could update all these imports if we want in another PR